content/events/kick-off.yaml

```diff
@@ -19,30 +19,30 @@
     url: https://techlabsorg.typeform.com/to/bEsJr5Gh
   resources: 
     # - title: "Meet: Kick-Off"
-    #   description: "The main room where we will meet. Passcode: 58008393"
+    #   description: "The main room where we will meet. Passcode: "
     #   type: meeting
-    #   url: https://uni-potsdam.zoom.us/j/68970635463
+    #   url: 
     # - title: "Meet: DS"
     #   description: Break-Out Room for Data Science
     #   type: meeting
-    #   url: https://meet.google.com/ept-mmdn-nyx
+    #   url: 
     # - title: "Meet: AI"
     #   description: Break-Out Room for Artificial Intelligence
     #   type: meeting
-    #   url: https://meet.google.com/vam-djgi-iof
+    #   url: 
     # - title: "Meet: WebDev"
     #   description: Break-Out Room for Web Development
     #   type: meeting
-    #   url: https://meet.google.com/prb-ifih-qnt
+    #   url: 
     # - title: "Meet: UX"
     #   description: Break-Out Room for UX Design
     #   type: meeting
-    #   url: https://meet.google.com/aep-cocb-wyq
+    #   url: 
     - title: "Slides"
       description: The presentation we showed you during the Kick-Off
       type: slides
-      url: https://drive.google.com/file/d/1XBI-2QSYIQ2mLOE3A0W-qPRLYkJEWACZ/view?usp=sharing
+      url: 
```

Comment:
instead of removing the url it would make sense to comment out the whole object. Otherwise we would have a slide & recording button on the card without any content behind it.

Reply:
Done